template.go

```diff
@@ -35,6 +37,18 @@ func siacoinString(siacoins types.Currency) string {
 	return fmt.Sprintf("%f %s", coins, coinPostfixes[i])
 }
 
+func byteString(numBytes uint64) string {
```

Comment:
also fitting this pattern are rateString and siacoinString, which are very similar. Maybe byteNumString?